tools/reference-workspace/src/recordings/uiTestScenarios/executions/46588b57054306cfdb4d67c9ce0f0362/scenarios.json

```diff
@@ -383,5 +383,48 @@
         },
         "scenario": "single measure",
         "vis": "PyramidChart"
+    },
+    {
+        "n2oMap": { "m_aangOxLSeztu": "m_aangOxLSeztu" },
```

Comment:
Thanks for the storybook tests, please add also waterfall chart to the general scenario and add one generic line to this test https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardTigerWithCharts.spec.ts

Reply:
I updated this. Thank you!